statistics/table.go

```diff
@@ -1378,3 +1386,18 @@ func CheckAnalyzeVerOnTable(tbl *Table, version *int) bool {
 	// This table has no statistics yet. We can directly return true.
 	return true
 }
+
+// recordUsedItemStatsStatus only records un-FullLoad item load status during user query
+func recordUsedItemStatsStatus(sctx sessionctx.Context, loadStatus StatsLoadedStatus,
+	tableID, id int64, isIndex bool) {
+	if sctx.GetSessionVars().InRestrictedSQL || loadStatus.IsFullLoad() {
```

Comment:
I think we don't need to filter internal SQLs here.

Reply:
updated.